### ISSUE TITLE
Workshop 404

### ISSUE THREAD

**Issue by owhc** [IBM/istio101#183]
Workshop doesnot exist. 404 page not found.

**Comment by zet809** [comment]
Seems more information are here: https://github.com/IBM/istio101/tree/master/docs

**Comment by owhc** [comment]
No.. that one is only lab instruction. In section : https://github.com/IBM/istio101/tree/master/docs/exercise-1#clone-the-lab-repo , It asked users clone fro repo , but workshop is missing.

**Comment by zet809** [comment]
In commit https://github.com/IBM/istio101/commit/aed79fc9de89e51c8d49fdb589bbfd722655f376, it seems `workshop` is changed to `docs`
So, go to docs
